src/strings/en-us.json

```diff
@@ -652,7 +652,7 @@
     "LabelH264Crf": "H264 encoding CRF:",
     "LabelH264EncodingPreset": "H264 encoding preset:",
     "LabelHardwareAccelerationType": "Hardware acceleration:",
-    "LabelHardwareAccelerationTypeHelp": "This is an experimental feature only available on supported systems.",
+    "LabelHardwareAccelerationTypeHelp": "Hardware acceleration requires additional configuration.",
```

Comment:
Should copy changes like this not be handled with weblate?

Reply:
What's going to be handled with Weblate? This is _source_ language...

Reply:
well, we keep doing something to break weblate. it isn't us making changes to these strings on git rather than weblate?

Reply:
Looks like source strings can be edited with Weblate unless I am missing something.

![Screenshot_2020-02-18 Jellyfin Jellyfin Web - English (United States)](https://user-images.githubusercontent.com/3450688/74749750-a0582980-5238-11ea-9805-3680d78fc8c7.png)

Just trying to avoid conflicts due to changes being made in Weblate **and** via PRs although that is probably less likely to happen in source language files.

Reply:
Oh I thought we had the source languages locked. This shouldn't cause any issues but I'll make an account over there just in case.